### PR TITLE
findutils v4.6.0

### PR DESCRIPTION
**Pull request by karl616**
I realized that there are bioconda tools depending on findutils. This becomes apparent first when then are containerized. The package triggering this for me was cromwell, which makes use of the `find -empty` flag that is not present in the busybox implementation

### CODE REVIEW

**Comment by karl616** [comment]
Disabled `make check` as there are some unfixed bugs compared to today. This is the latest release and it is three years old.

File: recipes/findutils/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  script:
+    - ./configure --prefix="{{ PREFIX }}"
+    - make -j${NUM_CPUS}
```
Comment (isuruf) [inline recipes/findutils/meta.yaml]: Use CPU_COUNT

Comment (karl616) [reply recipes/findutils/meta.yaml]: Done

File: recipes/findutils/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  url: https://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  script:
+    - ./configure --prefix="{{ PREFIX }}"
+    - make -j${CPU_COUNT}
+    - make install
```
Comment (ocefpaf) [inline recipes/findutils/meta.yaml]: Is there a `make check` we can run here?

Also note that as the build script gets bigger and more complex we recommend to use a separate `build.sh` file.

Comment (karl616) [reply recipes/findutils/meta.yaml]: I started from the gawk recipe, and it was setup like this. I can move this to a build script.

There is a `make check`. The latest findutils release is three years old and at least one test contains a bug. I was able to find a workaround for this (disabling). A second test fails during the travis-ci build and it is currently not part of the test set.

**Comment by karl616** [comment]
Hi @isuruf, hi @ocefpaf,
after a lot of back and forth I found a way through the checks. In the end I think it came out better this way as I ended up including patches from the Fedora project weeding out some bugs and adding a bit of functionality. I wasn't able to solve the ahistorical time formating tests for osx and ended up disabling them as I don't have access to such a system. This isn't major and when I asked the findutils devs, one recommendation was to disable the gnulib tests due to them being three years old.

When you have time, I would very much appreciate a second look.
Best,
Karl

**Comment by bgruening** [comment]
@karl616 thanks a lot looks good to me!

**Comment by karl616** [comment]
@bgruening Thank you!

**Pull request merged**